cime_config/acme/machines/config_machines.xml

```diff
@@ -574,7 +574,7 @@
 
 <machine MACH="blues">
          <DESC>ANL/LCRC Linux Cluster</DESC>
-         <NODENAME_REGEX>blogin</NODENAME_REGEX>
+         <NODENAME_REGEX>(blogin|^b\d)</NODENAME_REGEX>
```

Comment:
This is change is needed so that the machine is recognized in interactive sessions.